tensorboard/components/tf_card_heading/tf-card-heading.html

```diff
@@ -168,6 +171,11 @@
         this.$.descriptionDialog.positionTarget = e.target;
         this.$.descriptionDialog.toggle();
       },
+
+      // Break the string at natural points, including commas, equals, and slashes
+      _breakString: function(originalString) {
+        return originalString.replace(/([\/=\-_,])/g, "$1<wbr>");
```

Comment:
This could be a security risk. See [this](https://github.com/google/closure-library/blob/master/closure/goog/html/sanitizer/htmlsanitizer.js). Also:

- `import HtmlSanitizer from 'goog:goog.html.sanitizer.HtmlSanitizer';`
- `HtmlSanitizer.sanitize(...)`
- `@io_bazel_rules_closure//closure/library`.

Reply:
Yeah, you're right... should have thought more about that, this was just code I copy-pasted from tf-runs-selector.html as introduced in 3113df49e1a3607fc8312c2ebbdedc4b7c28148c, but I did think the innerHtml "workaround" was a bit funny.

There are a few other places using the innerHtml stuff that seem like they could also be problematic, e.g. here there's a homegrown HTML sanitizer function that while perhaps sufficient for that case would be better off sticking to HtmlSanitizer so nobody copies it into a place where it's inappropriate:
https://github.com/tensorflow/tensorboard/blob/68525d112aaf2fa0c61a1ed42a311c7a1f55dc1f/tensorboard/components/vz_line_chart/vz-line-chart.ts#L374

I can do a separate PR to fix all those spots to use HtmlSanitizer and then come back and update this one.